sdks/python/src/opik/opik_context.py

```diff
@@ -104,6 +106,7 @@ def update_current_span(
         "provider": provider,
         "total_cost": total_cost,
         "attachments": attachments,
+        "error_info": error_info,
```

Comment:
Let's add one unit test to tests/unit/decorator/test_tracker_outputs.py
The case would be:
1. update_current_span(error_info=...) is called
2. after that an error is raised from the tracked function
3. error info from the exception overrides error_info from the update call
So that we have this interaction described in tests :)

It would also be nice to update `test_track__span_and_trace_updated_via_opik_context` (or add a similar test) to include this field in a happy flow check.